modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -13463,6 +13447,33 @@ else if (type == Type.ONE_TO_MANY) {
 		return Type.MANY_TO_MANY;
 	}
 
+	private JSONObject _getVirtualInstanceJSONObject(String portalInstanceId)
```

Comment:
This is weird, the method is called `_getVirtualInstanceJSONObject` but the parameter `portalInstanceId`

Reply:
I would say, rename to `_getPortalInstanceJSONObject`, as the API is called portal instances

Reply:
I added a commit to rename some fields!